doc/config.md

```diff
@@ -7,11 +7,13 @@ The volcano device plugin has a number of options that can be configured. These
 | Flag                     | Envvar                  | Default Value   |
 |--------------------------|-------------------------|-----------------|
 | `--gpu-strategy`         | `$GPU_STRATEGY`         | `"share"`       |
+| `--gpu-memory-factor`    | `$GPU_MEMORY_FACTOR`    | `1`             |
 | `--config-file`          | `$CONFIG_FILE`          | `""`            |
 
 when starting volcano-device-plugin.yml, users can specify these parameters by adding args to the container 'volcano-device-plugin'.
 For example: 
  - args: ["--gpu-strategy=number"] will let device plugin using the gpu-number strategy
+ - args: ["--gpu-strategy=share","--gpu-memory-factor=10"] will let device plugin using the gpu-share strategy, and memory factor is 10MB
```

Comment:
Are there some limit about the parameter `gpu-memory-factor`? For example, could it be a decimal？

Reply:
This is not recommended generally, the most common setting is 1MiB / 1 GiB

Reply:
Yes, so I mean it should be highlight.